### ISSUE TITLE
error in C code generated from call to read_file_array with the generic type of u8

### ISSUE THREAD

**Issue by vyrus001** [vlang/v#15725]
**V doctor:**
```
OS: windows, Microsoft Windows 10 Pro v19044 64-bit
Processor: 64 cpus, 64bit, little endian, AMD Ryzen Threadripper 2990WX 32-Core Processor  

CC version: Error: exec failed (CreateProcess) with code 2: The system cannot find the file specified.

 cmd: cc --version

getwd: C:\Users\vyrus\proj\crackmagicexec
vmodules: C:\Users\vyrus\proj\v\mylibs
vroot: C:\Users\vyrus\tools\v
vexe: C:\Users\vyrus\tools\v\v.exe
vexe mtime: 2022-09-11 06:29:55
is vroot writable: true
is vmodules writable: true
V full version: V 0.3.1 ca99a1d.7eb3678

Git version: git version 2.37.3.windows.1
Git vroot status: weekly.2022.36-23-g7eb36789
.git/config present: true
thirdparty/tcc status: thirdparty-windows-amd64 125e21e2

```

**What did you do?**
`v -g -o vdbg cmd/v && vdbg .\test.vsh`
```v
test_file_name := 'test_file'
write_file_array(test_file_name, [u8(0xff), 0xff, 0xff, 0xff])?
defer {rm(test_file_name) or { panic(error) }}
read_file_array<u8>(test_file_name)
```

**What did you expect to see?**

nothing

**What did you see instead?**
```
==================
C:/Users/vyrus/AppData/Local/Temp/v_0/test.468243329413189086.tmp.c:17464: warning: implicit declaration of function 'os__read_file_array_T_u8'
tcc: error: undefined symbol 'os__read_file_array_T_u8'
...
==================
(Use `v -cg` to print the entire error message)

builder error: 
==================
C error. This should never happen.

This is a compiler bug, please report it using `v bug file.v`.

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang

```

**Comment by spytheman** [comment]
A temporary workaround seems to be to `import os` explicitly, so it is probably related to vscript.
```v
import os
test_file_name := 'test_file'
os.write_file_array(test_file_name, [u8(0xff), 0xff, 0xff, 0xff])?
defer {os.rm(test_file_name) or { panic(error) }}
os.read_file_array<u8>(test_file_name)
```

**Comment by spytheman** [comment]
Fixed in e03ca20 .
